Makefile

```diff
@@ -48,13 +51,12 @@ ETHEREUM_PACKAGE_REVISION := c7952d75d72159d03aec423b46797df2ded11f99
 # Shallow clones can't specify a single revision, but at least we avoid working
 # the whole history by making it shallow since a given date (one day before our
 # target revision).
-ETHEREUM_PACKAGE_SHALLOW_SINCE := 2024-08-23
```

Comment:
why remove this?

Reply:
It was causing the clone to fail on my side.